.circleci/config.yml

```diff
@@ -7,10 +7,10 @@ orbs:
   aws-ecr: circleci/aws-ecr@9.0.2 # this orb doesn't support OIDC v2, so we use aws-cli to authenticate
 
 executors:
-  python-build-image-executor:
-    resource_class: small
-    docker:
-      - image: cimg/python:3.12.1
+  # python-build-image-executor:
```

Comment:
Rather than comment out the python CI stuff, I think we should delete it. But it's bad to have code in the repo that's not being exercised by CI, so I think if we delete the python CI stuff we should delete the python code from the legacy folder (we were only holding on to it in case the transition to Ruby went horribly wrong)

Reply:
will do a separate PR after this is merged. just to have something clean to revert if ever needed